meerkat_abacus/data_management.py

```diff
@@ -804,7 +816,12 @@ def create_links(data_type, input_conditions, table, session, conn,
                     column, condition = link["to_condition"].split(":")
                     conditions.append(
                         link_alias.data[column].astext == condition)
+                if link.get("from_condition"):
+                    column, condition = link["from_condition"].split(":")
+                    conditions.append(
+                        link_alias.data[column].astext == condition)
```

Comment:
I don't like this broken line here in 821. You could fit in line length restriction by extracting the condition to local variable.
The same amount of code and much better readability. Also the 2nd return of split is not a `condition` yet. It's looks like `expected_text` or `expected_value`. I'd write:
```python
column, expected_text = link.split()
condition = link_alias.data.astext == expected_text
conditions.append(condition)
```
This code takes your hand and walks you through what exactly is going on here.